docs/_articles/en/custom-rules/author.md

```diff
@@ -20,7 +20,7 @@ If you are using an IDE, add ```$PMD_BIN_HOME/lib/*``` to its CLASSPATH. To comp
 
 ```$ javac -cp ".:$PMD_BIN_HOME/lib/*" /path/to/your/Rule.java```
 
-Use only [version 6.22.0](https://github.com/pmd/pmd/releases/download/pmd_releases%2F6.22.0/pmd-bin-6.22.0.zip) of PMD for writing the custom rules. 
+Use only [version {{ site.data.versions.pmd }}](https://github.com/pmd/pmd/releases/download/pmd_releases%2F6.22.0/pmd-bin-6.22.0.zip) of PMD for writing the custom rules. 
```

Comment:
Any suggestions on the hard coded link? if we upgrade to 6.27 next release, this will be wrong.

Reply:
The link is usually standardized. We'll need to replace the `6.22.0` part with `{{ site.data.versions.pmd }}`